instrumentation-genai/opentelemetry-instrumentation-openai-v2/pyproject.toml

```diff
@@ -25,9 +25,9 @@ classifiers = [
   "Programming Language :: Python :: 3.12",
 ]
 dependencies = [
-  "opentelemetry-api ~= 1.12",
-  "opentelemetry-instrumentation ~= 0.48b0",
-  "opentelemetry-semantic-conventions ~= 0.48b0"
+  "opentelemetry-api ~= 1.28",
+  "opentelemetry-instrumentation ~= 0.49b0",
+  "opentelemetry-semantic-conventions ~= 0.49b0"
```

Comment:
Should we pin exact versions to avoid mismatching dependencies?

Reply:
we should allow any version after a certain one - otherwise we can't combine multiple instrumentations:

e.g. openai-v2 would require 0.49, azure resource detector would require 0.43 and they could never work together. Or am I missing something about python dependency management?